go/ct/spc/specification.go

```diff
@@ -1402,6 +1402,26 @@ func getAllRules() []Rule {
 		},
 	})...)
 
+	// notEmpty
+	rules = append(rules, rulesFor(instruction{
+		op:        vm.EXTCODEHASH,
+		name:      "_not_empty",
+		staticGas: 2600, // highest possible cost.
```

Comment:
This static price assumes a cold address, right? But the condition is not specifying this.

It is very unlikely that a random address is indeed warm, so tests may pass. But conceptually, I think this rule is contradicting other rules.

Reply:
solved by adding cross product of warm/cold, non/empty and revisions.